betterlockscreen

```diff
@@ -20,6 +20,7 @@ init_config () {
     pixel_scale=10,1000
     solid_color=333333
     description=""
+    quiet=""
```

Comment:
Could you please also adjust the default value to false as it seems to be feature toggle? :)

Reply:
fixed